wikibase/1.30/bundle/extra-preinstall-runtime.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+/wait-for-it.sh $MW_ELASTIC_HOST:9200 -t 60
```

Comment:
I wonder if the port wants to be optionally passed in as an env var?
Could make sense for people using externals elastic clusters and the wikibase bundle image?

Reply:
I think it does; I'll add that